cypress/integration/remote-storage/configurations/crud.spec.js

```diff
@@ -0,0 +1,127 @@
+import { bigtestGlobals } from '@bigtest/globals';
+import { Link } from '@bigtest/interactor';
+import {
+  Button,
+  KeyValue,
+  Modal,
+  MultiColumnList,
+  MultiColumnListCell,
+  Pane,
+  Select,
+  TextField
+} from '../../../../interactors';
+
+bigtestGlobals.defaultInteractorTimeout = 10000;
+
+describe('making CRUD', () => {
+  before(() => {
+    cy.visit('/');
+
+    cy.login('diku_admin', 'admin');
+
+    // We rely here on Settings routing,
+    // but navigate inside our own module-under-test using UI
+    cy.visit('/settings/remote-storage');
+    cy.do(Link('Configurations').click());
+  });
+
+  it('should provide happy path', function () {
+    const NAME = 'Test name 1';
+    const NAME_EDITED = 'Test name 2 - edited';
+
+    const PROVIDER = 'Dematic EMS';
+    const PROVIDER_CODE = 'DEMATIC_EMS';
+
+    // Create
+    cy.do(Button('+ New').click())
+      .expect([
+        Pane('Create configuration').exists(),
+        TextField('Remote storage name\n*').is({ focused: true, value: '' }),
+        Select('Provider name\n*').has({ value: '' }),
+        Button('Cancel').exists(),
+        Button('Save & close').exists(),
+      ])
+
+      .do([
+        TextField('Remote storage name\n*').fillIn(NAME),
+        Select('Provider name\n*').choose(PROVIDER),
+        Button('Save & close').click(),
+      ])
+      .expect([
+        Modal('Create configuration').exists(),
```

Comment:
This is redundant since you're asserting that both the save and cancel buttons exist within this modal, you will get an error message specifically highlighting the absence of the 'create configuration' modal if it does not exist. (it won't even look for the save and cancel buttons because it will short circuit)

Since the next block of actions uses the save button, you don't need to verify that it exists, or wait to proceed until it exists. The act of using it will wait wait until it exists, is visible, and is enabled, and if it is not all of those things you will get a precise error message detailing why. Also, if it cannot be found it will look for other buttons on the page and present them as possible alternatives.